### PR TITLE
Sysbuild/partition manager

### PR DESCRIPTION
**Pull request by tejlmand**
To be filled.

### CODE REVIEW

File: Kconfig.zephyr [hunk 1]
```diff
@@ -4,6 +4,10 @@
 # Copyright (c) 2016 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+# Ensure that any setting defined by sysbuild takes precedence.
+# For eample if sysbuild defines that MCUboot is enabled, then it's enabled
```
Comment (nordicjm) [inline Kconfig.zephyr:8]:
```suggestion
# For example if sysbuild defines that MCUboot is enabled, then it's enabled
```

File: share/sysbuild/CMakeLists.txt [hunk 2]
```diff
@@ -117,6 +117,13 @@ endwhile()
 
 sysbuild_module_call(PRE_CMAKE MODULES ${SYSBUILD_MODULE_NAMES} IMAGES ${IMAGES})
 foreach(image ${IMAGES})
+  # Create an sysbuild controlled Kconfig for the image.
```
Comment (nordicjm) [inline share/sysbuild/CMakeLists.txt]:
```suggestion
  # Create a sysbuild controlled Kconfig for the image.
```

File: CMakeLists.txt [hunk 3]
```diff
@@ -1711,6 +1716,7 @@ if(CONFIG_BUILD_OUTPUT_EXE)
     post_build_byproducts
     ${KERNEL_EXE_NAME}
     )
+  set(BYPRODUCT_KERNEL_EXE_NAME "${PROJECT_BINARY_DIR}/${PROJECT_BINARY_DIR}/${KERNEL_EXE_NAME}" CACHE FILEPATH "Kernel exe file" FORCE)
```
Comment (nordicjm) [inline CMakeLists.txt:1719]: this doesn't look right

**Comment by de-nordic** [comment]
Needs rebase

File: Kconfig.zephyr (hunk 1 above)
Comment (tejlmand) [reply Kconfig.zephyr]: Outdated.
Now using  https://github.com/zephyrproject-rtos/zephyr/pull/55871 instead.

File: share/sysbuild/CMakeLists.txt (hunk 2 above)
Comment (tejlmand) [reply share/sysbuild/CMakeLists.txt]: Outdated.
Now using  https://github.com/zephyrproject-rtos/zephyr/pull/55871 instead.

**Review by nordicjm — APPROVED** [review]

**Comment by 57300** [comment]
Could you cherry-pick https://github.com/zephyrproject-rtos/zephyr/commit/65c9058ed4769cbb4957c4d111e96252b29ffb33 to prevent some MCUboot build errors?

**Comment by tejlmand** [comment]
> Could you cherry-pick [zephyrproject-rtos/zephyr@65c9058](https://github.com/zephyrproject-rtos/zephyr/commit/65c9058ed4769cbb4957c4d111e96252b29ffb33) to prevent some MCUboot build errors?

will do

**Review by 57300 — APPROVED** [review]

**Pull request merged**